chipsec/helper/win/win32helper.py

```diff
@@ -353,6 +353,8 @@ def create(self, start_driver):
         if logger().VERBOSE: logger().log( "[helper] service control manager opened (handle = 0x%08x)" % hscm )
 
         driver_path = os.path.join( chipsec.file.get_main_dir(), "chipsec", "helper", "win", self.win_ver, DRIVER_FILE_NAME )
+        if not os.path.isfile( driver_path ):
+            driver_path = os.path.join("C:\\Windows\\System32\\drivers\\RwDrv.sys"
```

Comment:
Hum. Good point. I think the idea was to allow someone to put the chipsec driver in system32/drivers as an alternative to in the chipsec directory. Would you rather see a commit that removes this or one that fixes it to look for the chipsec driver under system32?